runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -99,6 +99,16 @@ default Color getTargetColor()
 		return Color.RED;
 	}
 
+	@ConfigItem(
+			position = 7,
```

Comment:
Fix the indentation (just import XML here: https://github.com/runelite/runelite/wiki/Code-Conventions to IntelliJ)